portfolio/src/main/webapp/index.html

```diff
@@ -64,9 +67,37 @@ <h2> PROJECTS</h2>
 
       <button onclick="addRandomQuoteOrFact()"> facts + quotes </button>
       <div id = "greeting-container"></div>
-
-      <button onclick="hello()"> hi </button>
-      <div id = "response"></div>
+      <div class ="commentForm">
+        <h2 class="contactSubtitle">Leave a comment!</h2>
+        <form action="/data" method="POST">
+          <label for ="userName" style="display:block: margin-top: .5em";> Name: </label>
+            <div id = "nameHolder">
+              <input type="text" id="userName" name="userName" maxlength="35" placeholder="Your name...">
+            </div>
+         <label for="comment" style="display:block; margin-top: .5em;">Comment:</label>
+            <div id="textareaHolder">
+            <textarea id="comment" name="comment" rows="4" cols="50" maxlength="600" placeholder="Add a comment..."></textarea>
+            </div>
+          <button onclick="hello()"> post </button>
```

Comment:
if you just keep onload="hello()" and remove this onclick="hello()" part, does it work as expected?

Reply:
no. it causes errors unless i have both.